rate_counter/rate/views.py

```diff
@@ -1,3 +1,161 @@
-from django.shortcuts import render
+from django.shortcuts import render, HttpResponseRedirect, get_object_or_404, get_list_or_404
+from .models import Institute, Subject, StudentGroup, Table, TableRow
+from .forms import TableForm
 
 # Create your views here.
+
+
+def create_table(request):
+    """
+    creation of new table
+    """
+    form = TableForm(request.POST or None)
+    if form.is_valid():
+        instance = form.save(commit=False)
+        instance.save()
+        return HttpResponseRedirect("/rate/detail/%s/" % instance.id)
```

Comment:
Better to use 'format' method. % is a C-style formatting, I think it was removed in Python 3